Frends.Template/Frends.Echo.Execute/Definitions/Options.cs

```diff
@@ -0,0 +1,29 @@
+﻿// <copyright file="Options.cs" company="Frends">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace Frends.Echo.Execute.Definitions;
+
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+/// <summary>
+/// Parameters class usually contains parameters that are required.
+/// </summary>
+public class Options
+{
+    /// <summary>
+    /// Number of times input is repeated.
+    /// </summary>
+    /// <example>Example of the parameter value.</example>
```

Comment:
Well I thought that because this is a template, these values should always be replaced when using the template. That's why I wanted to use this text to indicate that this needs to be changed.